tutorials/notebooks/example_keras_effdet_lite0.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
General comments:
I think the headlines in the tutorial should be shorted, and instead detail a bit at the start of each paragraph. 
Also, I think it should have more comments in the code snippets (especially in functions doc and type hints).

There are small things that maybe should changed like sys.path.insert(0,"/content/local_mct") in the middle of imports, large sections could be fragmented into smaller sections.

Reply:
One more thing, just try to use GPU in the colab, to make sure it can be used as well (since we sometimes convert pytorch tensors to numpy in the tutorial)